spec/team_stats_spec.rb

```diff
@@ -25,4 +25,12 @@
     expect(@stat_tracker.team_info("1")["abbreviation"]).to eq("ATL")
     expect(@stat_tracker.team_info("1")["link"]).to eq("/api/v1/teams/1")
   end
+
+  it "#favorite_opponent" do
```

Comment:
I'm assuming here we're writing tests directly from the spec harness?  That works for me -- will give some thought as I work on the methods here, but also fine with this approach (and can't immediately think of another).